### PR TITLE
Fix a PHPStan false-positive with changing values in $GLOBALS

### PR DESCRIPTION
**Pull request by m-vo**
| Q                | A
| -----------------| ---
| Fixed issues     | -
| Docs PR or issue | -

Fixes the issues reported in the CI.

### CODE REVIEW

**Comment by m-vo** [comment]
Not sure if this is also a problem in 4.9.

File: phpstan.neon.dist [hunk 1]
```diff
@@ -36,6 +36,11 @@ parameters:
         - '#Call to an undefined method .*::transChoice\(\)\.#'
         - '#Trying to mock an undefined method transChoice\(\) on class Symfony\\Component\\Translation\\Translator\.#'
 
+        # Ignore PHPStan not being able to trace changes in $GLOBALS
+        -
+            message: '#Call to method PHPUnit\\Framework\\Assert::assertSame\(\) with .* will always evaluate to false\.#'
```
Comment (leofeyer) [inline phpstan.neon.dist]:
```suggestion
            message: '#Call to method PHPUnit\\Framework\\Assert::assertSame\(\) with \$GLOBALS.* will always evaluate to false\.#'
```

Limiting the regex will prevent false positives that are not related to `$GLOBALS`.

Comment (m-vo) [reply phpstan.neon.dist:41]: Should we also drop the `path` then?

Comment (m-vo) [reply phpstan.neon.dist]: This unfortunately won't work as the message looks like this:
```
Call to method PHPUnit\Framework\Assert::assertSame() with array(array('test.listener.a', 'onGetPageLayout'), array('test.listener.c', 'onGetPageLayout'), array('test.listener.b',            
         'onGetPageLayout')) and array(array('test.listener.c', 'onGetPageLayout')) will always evaluate to false.   
```

Comment (leofeyer) [reply phpstan.neon.dist]: I see. 🤷🏻‍♂️

**Review by leofeyer — CHANGES_REQUESTED** [review]
If this really only occurs in one single place, we should use `@phpstan-ignore-next-line` instead of a global exception rule.

**Review by bytehead — APPROVED** [review]

**Comment by m-vo** [comment]
> If this really only occurs in one single place, we should use `@phpstan-ignore-next-line` instead of a global exception rule.

It's more than one place. :grimacing:

**Comment by leofeyer** [comment]
Thank you @m-vo.

**Comment by m-vo** [comment]
Just noticed that the issue from the `XliffFileLoaderTest` also happens in 4.9 :disappointed: 

@leofeyer Shall I make a second PR?

**Comment by leofeyer** [comment]
Yes please.

**Comment by m-vo** [comment]
see https://github.com/contao/contao/pull/2793
